python/RELEASING.md

```diff
@@ -0,0 +1,15 @@
+# How to make a release
```

Comment:
This here should be using the auto-copy from polyglot-release.

See an example in one of the ruby projects I manage here: https://github.com/cucumber/cucumber-ruby/blob/main/RELEASING.md

Which links to: https://github.com/cucumber/.github/blob/main/RELEASING.md

This will tell you how to get a few things configured and setup, specifically for some of the internal toolings.

If you need more dedicated help, join us on slack @ cucumberBDD.slack.com